### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by MarcoOquendoC**
# [Portfolio: details popup window](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md)

## Hi good reviewer,

Have a good day bear

### These are the [changes](https://github.com/MarcoOquendoC/Portfolio/tree/details-popup-window) proposed to the [Portfolio](https://github.com/MarcoOquendoC/Portfolio) project:

[README.md]()
[index.html](https://github.com/MarcoOquendoC/Portfolio/blob/mobile-menu/index.html)
[styles.css](https://github.com/MarcoOquendoC/Portfolio/blob/mobile-menu/styles.css)
[script.js](https://github.com/MarcoOquendoC/Portfolio/blob/mobile-menu/script.js)

#### Project requirements
- Implement the following interactions:
  - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  - When the user clicks (or taps) the close (X) button, the popup disappears.

- You should implement popups for both mobile and desktop screen sizes:

- In order to associate each project with the popup window details. Use a JavaScript array to store all of the information for all projects.
  - For each project. Store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
  - Update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
  - Implement the popup window.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -46,113 +46,32 @@ <h2>
         My Recent Works
       </h2>
```
Comment (MeqdamAlqudah) [inline index.html:47]: - [ ] Please fix linter errors 
image

File: index.html [hunk 2]
```diff
@@ -46,113 +46,32 @@ <h2>
...
       <hr>
-      <div class="work-grid">
-        <article class="work-card">
-          <div class="work-card-img">
-          </div>
-          <div class="work-card-content">
```
Comment (MeqdamAlqudah) [inline index.html:53]: - [ ] Please make sure that the popup is responsive 
image

File: index.html [hunk 3]
```diff
@@ -46,113 +46,32 @@ <h2>
...
-            <h3> 
-              Multi-Post Stories Gain+Glory 
-            </h3>
-            <ul class="tags">
```
Comment (MeqdamAlqudah) [inline index.html:57]: - [ ] Please make  sure that your design matches Figma design 

| Your design | Figma design |
| ---------------- | ------------------ |
| image | image |

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @MarcoOquendoC , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MarcoOquendoC** [comment]
# Hi @MeqdamAlqudah,

## Thank you for your review.

### The changes on the repository are listed below:
- No linter errors.
- Improve responsiveness.
- Change styles to match the figma design.

**Review by MeqdamAlqudah — APPROVED** [review]
@MarcoOquendoC , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**